boot/initramfs_test.go

```diff
@@ -620,10 +620,10 @@ func (s *initramfsSuite) TestInitramfsRunModeSelectSnapsToMount(c *C) {
 			c.Assert(err, IsNil, comment)
 
 			if t.expRebootPanic != "" {
-				f := func() { boot.InitramfsRunModeSelectSnapsToMount(t.typs, m) }
+				f := func() { boot.InitramfsRunModeSelectSnapsToMount(t.typs, m, boot.InitramfsWritableDir) }
```

Comment:
should this test use some other temp directory instead

Reply:
Tests added too